docs/en/academic/custom-templates.md

```diff
@@ -48,6 +48,14 @@ $body$
 
 Pandoc will replace this variable with the parsed contents of your Markdown file(s). If you leave it out, your content will not appear in the output file.
 
+## Activate your template
+
+In order to put your template to work, you must point Zettlr to it, via the PDF Defaults File in the Assets Manager. Navigate to the Assets Manager from Zettlr's Menu and select 'PDF' from the list of configurations ('Defaults Files') on the left. Next add `template: ` to the bottom of the script. Note the space after the colon. Now Zettlr needs to know the path to your new template file. Locate your template file, which if you've created it in Zettlr, you may find by right-clicking on the file in Zettlr's file manager and select 'Show File'. Note that the name of the LaTeX file must end in `.tex`. Once you've found the file, you need to find it's location in your computer's directory structure – which is the file's 'path' or 'pathname'. On macOS, you can find the file's path by right-clicking the file in the Finder and hit the 'Option' key, which will change the 'Copy file' menu item to 'Copy as Pathname'. Copy the pathname and paste it into the PDF Default File thus: `template: /path/to/your/template.tex`. 
```

Comment:
It is not a "script" but a configuration file, so I would make that more clear. It is also good that you mention that you need the space between the colon and the value.

Other than that I would try to find easy ways of explaining how to find the path to those files that works for all operating system (involves a little bit of googling) and then maybe do a list like:

```markdown
You need to find the path to your template:

* On Windows, do XYZ
* On macOS, do ABC
* On Linux, do DEF
```